api/src/main/resources/application.conf

```diff
@@ -20,6 +20,12 @@ akka {
   cluster {
     failure-detector.threshold = 12 //Increase value for AWS environments
     metrics.enabled = off
+
+    retry-unsuccessful-join-after = 20s
+
+    //DON'T USE IN PRODUCTION!!!!!
```

Comment:
Auto downing of nodes is not recommended in production systems due to the possibility of network failures that form a partition in the cluster. It's useful in development so that the cluster lifecycle (joining, unreachable, leaving, etc. ) can be tested